RESTResource/RESTResource.js

```diff
@@ -324,6 +324,17 @@ export default class RESTResource {
         options.params = options.params(parsedQuery, _.get(props, ['match', 'params']), mockProps(state, this.module, props.dataKey, this.logger).resources, this.logger, props);
       }
 
+      // headers
```

Comment:
This block of code looks fine.

My only hesitation is whether it would be better to abstract out the repeated block into a function that can be invoked to do this for both `params` and `headers` — also opening the way for it to be trivial to do the same for other parts of the `options` object.

Reply:
Good point, I've now updated the PR accordingly.

Reply:
Nice!